modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase

```diff
@@ -2466,24 +2464,32 @@ definition {
 
 		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject287");
 
-		for (var number : list "1,2,3,4,5,6,7,8") {
+		for (var number : list "1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41") {
 			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
 				fieldName = "customObjectField",
 				objectName = "CustomObject287",
-				value = "Test text ${number}");
+				value = "Text ${number} test");
 		}
 
 		ObjectAdmin.goToCustomObject(objectName = "CustomObject287");
 
-		ObjectPortlet.changePagination(itemsPerPage = "4 Items");
+		ObjectPortlet.changePagination(itemsPerPage = "20 Items");
 
-		for (var number : list "1,2,3,4") {
-			ObjectPortlet.viewEntry(entry = "Test text ${number}");
+		for (var number : list "1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20") {
+			ObjectPortlet.viewEntry(entry = "Text ${number} test");
 		}
 
-		for (var number : list "5,6,7,8") {
-			ObjectPortlet.assertEntryNotPresent(entry = "Test text ${number}");
+		for (var number : list "21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41") {
+			ObjectPortlet.assertEntryNotPresent(entry = "Text ${number} test");
 		}
```

Comment:
here, instead of checking everything, we can just validate the absence of the 21 element `ObjectPortlet.assertEntryNotPresent(entry = "Text 21");`